src/app/components/options-sidebar/time-clock/time-clock.component.html

```diff
@@ -1,89 +1,79 @@
-<div class="text-center mt-3">
-
-  <div *ngIf="showAlertEnterTecnology" class="alert alert-danger" role="alert">
-    Field technology is requiered. Enter this field for clock out.
-  </div>
-
-  <div class="card">
-    <div class="card-header">
-      <div class="row">
-        <div class="col-2 text-left">
-          <strong>Time clock</strong>
-        </div>
-        <div class="col-10 text-right">
-          <i class="far fa-question-circle"></i>
+<div class="text-center mt-5">
+    <div class="card">
```

Comment:
Please, fix the indentation, remember that we are using an indentation based in 2 spaces.

Reply:
done